uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/manage-leave-requests.component.js

```diff
@@ -289,10 +269,12 @@ define([
           ]);
         })
         .then(function () {
-          /*
-           * If the status filter is not set to "All" and
+          /**
```

Comment:
According to the styleguide we decided to use `//` for non documentation comments.